.github/workflows/labeler.yml

```diff
@@ -1,10 +1,10 @@
 # https://github.com/marketplace/actions/labeler
-name: "Pull Request Labeler"
+name: Add labels
 on:
 - pull_request_target
 
 jobs:
-  triage:
+  labeler:
```

Comment:
Draga and I have been discussing removing this action altogether. I think those labels ("qt" "tests") are pretty useless?

Reply:
Maybe useless for release notes and such, but might be useful for searching PRs and similar. I don't think it hurts :shrug:

Reply:
The question is whether anyone has ever used them to search for PRs. 😂

Reply:
I have!

Reply:
Well I do this more in other projects, but I think they can be useful when done well!